### PR TITLE
GEOPY-1552: Crash on joint survey with apparent resistivity

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-1552 - Crash on joint survey with apparent resistivity**

### CODE REVIEW

**Comment by domfournier** [comment]
@sebhmg Any idea why imports would fail for pylint?

File: simpeg_drivers/joint/joint_surveys/driver.py [hunk 1]
```diff
@@ -16,17 +16,14 @@
...
 
-# pylint: disable=unexpected-keyword-arg, no-value-for-parameter
-
 from __future__ import annotations
 
 import numpy as np
 from geoh5py.shared.utils import fetch_active_workspace
 from SimPEG import maps
 
-from simpeg_drivers.components.factories import DirectivesFactory
```
Comment (sebhmg) [inline simpeg_drivers/joint/joint_surveys/driver.py:27]: not sure if relative import is mandatory here for pylint to work, but `__init__.py` probably was

Comment (domfournier) [reply simpeg_drivers/joint/joint_surveys/driver.py:27]: Yea I figured it out. Missing an init__ in there...

**Review by sebhmg — COMMENTED** [review]
no comment on the code here. I leave this review to @benk-mira

**Review by benk-mira — APPROVED** [review]

**Pull request merged**